src/web/layouts/CommentLayout.tsx

```diff
@@ -78,13 +90,17 @@ const StandardGrid = ({
                         1px /* Vertical grey border */
                         1fr /* Main content */
                         300px; /* Right Column */
+
                     grid-template-areas:
                         'title      border  headline    right-column'
                         'lines      border  headline    right-column'
                         'meta       border  standfirst  right-column'
                         'meta       border  media       right-column'
                         '.          border  body        right-column'
                         '.          border  .           right-column';
+
+                    /* Override grid-template-areas if showcase */
+                    ${display === Display.Showcase ? showcaseGrid : ''}
```

Comment:
I like this instead of the other conditional rendering we have in StandardLayout.tsx 

Just need to make sure we're covering all breakpoints?

Reply:
thanks, yeah I can throw up a PR to make those changes. Will test other breakpoints to make sure I havent missed anything